pkg/ebpf/conntrack/conntrack_client.go

```diff
@@ -151,6 +152,7 @@ func (c *conntrackClient) CleanupConntrackMap() {
 }
 
 func (c *conntrackClient) Cleanupv6ConntrackMap() {
+	c.logger.Info("Start cleanup of conntrack map")
```

Comment:
nit: `Start cleanup of conntrack map` --> `Check for any stale entries in the conntrack map`